bigbluebutton-html5/imports/ui/components/nav-bar/recording-indicator/component.jsx

```diff
@@ -48,25 +55,60 @@ const defaultProps = {
   amIModerator: false,
   record: false,
   recording: false,
-  title: '',
   time: 0,
 };
 
 class RecordingIndicator extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      time: (props.time ? props.time : 0),
+    };
+
+    this.incrementTime = this.incrementTime.bind(this);
+  }
+
+  componentDidUpdate() {
+    const { recording } = this.props;
+    if (!recording) {
+      clearInterval(this.interval);
+      this.interval = null;
+    } else if (this.interval === null) {
+      this.interval = setInterval(this.incrementTime, 1000);
+    }
+  }
+
+  incrementTime() {
+    const { time: propTime } = this.props;
+    const { time } = this.state;
+
+    if (propTime > time) {
+      this.setState({ time: propTime + 1 });
+    } else {
+      this.setState({ time: time + 1 });
+    }
+  }
+
   render() {
     const {
       record,
-      title,
       recording,
       mountModal,
-      time,
       amIModerator,
       intl,
       allowStartStopRecording,
     } = this.props;
 
+    const { time } = this.state;
     if (!record) return null;
 
+    if (!this.interval) {
```

Comment:
This should be changed to the following or the interval will be started before the recording is actually started.
```suggestion
    if (!this.interval && recording) {
```